parallel-install/README.md

```diff
@@ -13,28 +13,28 @@ Before starting the deployment or uninstallation process, you need to provide a
-| HelmTimeoutSeconds | `int` | `360` | Timeout for the underlying Helm client. |
-| BackoffInitialIntervalSeconds | `int` | `1` | Initial interval used for exponential backoff retry policy. |
-| BackoffMaxElapsedTimeSeconds | `int` | `30` | Maximum time used for exponential backoff retry policy. |
-| Log | `func(format string, v ...interface{})` | `fmt.Printf` | Function used for logging. To modify the logging behavior, set a custom logging function. For example, to disable any log output, provide an empty logging function implementation (`func(f string, v ...interface{}){}`). |
-| Profile | `string` | `evaluation` | Deployment profile. The possible values are: "evaluation", "production", "".  |
-| ComponentsListFile | `string` | `/kyma/components.yaml` | List of prerequisites and components used by the installer library. |
-| ResourcePath | `string` | `$GOPATH/src/github.com/kyma-project/kyma/resources` | Path to Kyma resources. |
-| InstallationResourcePath | `string` | `$GOPATH/src/github.com/kyma-project/kyma/installation/resources` | Path to Kyma installation resources. |
-| Version | `string` | `1.18.1` | The Kyma version. |
+| Parameter                     | Type                                    | Example value                                                     | Description                                                                                                                                                                                                                |
+| ----------------------------- | --------------------------------------- | ----------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| WorkersCount                  | `int`                                   | `4`                                                               | Number of parallel workers used for the `deploy` or `uninstall` operation.                                                                                                                                                 |
+| CancelTimeout                 | `time.Duration`                         | `900 * time.Second`                                               | Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client.                                                                                            |
+| QuitTimeout                   | `time.Duration`                         | `1200 * time.Second`                                              | Time after which the `deploy` or `uninstall` operation is aborted and returns an error to the user. Worker goroutines may still be working in the background. This value must be greater than the value for CancelTimeout. |
```

Comment:
This looks similar to an alpha command flag. Is this by design or are they related?

Reply:
the alpha command in the CLI uses the `parallel-install` library in hydroform to deploy `Kyma`